.github/release-drafter.yml

```diff
@@ -0,0 +1,59 @@
+name-template: '$RESOLVED_VERSION'
+tag-template: '$RESOLVED_VERSION'
+
+version-resolver:
+  major:
+    labels:
+      - 'major'
+  minor:
+    labels:
+      - 'minor'
+  patch:
+    labels:
+      - 'patch'
+  default: minor
+
+categories:
+  - title: 'Features'
+    label: 'enhancement'
+  - title: 'Bug Fixes'
+    label: 'bug'
+
+exclude-labels:
+  - 'skip'
+
+autolabeler:
+  - label: 'bug'
+    title:
+      - '/.*\[fix\].*/'
+  - label: 'patch'
+    title:
+      - '/.*\[fix\].*/'
+  - label: 'enhancement'
+    title:
+      - '/.*\[feat\].*/'
+  - label: 'minor'
+    title:
+      - '/.*\[feat\].*/'
+  - label: 'skip'
+    title:
+      - '/.*\[skip\].*/'
+  - label: 'major'
+    title:
+      - '/.*\[breaking\].*/'
+
+replacers:
+  - search: '/\[feat\]/g'
+    replace: ''
+  - search: '/\[fix\]/g'
+    replace: ''
+  - search: '/\[skip\]/g'
+    replace: ''
+  - search: '/\[breaking\]/g'
+    replace: ''
+
+template: |
+  # What's Changed
+
+  $CHANGES
```

Comment:
This is interesting, but where does the draft release notes show up? I've never noticed them on an open source project. I just checked out the ZIO repo (which uses release-drafter), and can't find them either.

I personally don't need it since github now has a 'Generate Release Notes' button. I don't mind the default format. You can remove this for now and we can loop back later.